build/policy/files.rego

```diff
@@ -58,6 +58,17 @@ deny[sprintf("Integration '%v' missing required attribute '%v'", [name, attr])]
 	some attr in (required - {key | some key, _ in item})
 }
 
+deny[sprintf("Integration '%v' references unknown software '%v' (i.e. not in 'software' object)", [name, software])] {
+	"docs/website/data/integrations.yaml" in filenames
+
+	file := yaml.unmarshal(integrations_file)
+	software_list := {software | file.software[software]}
+
+	some name, item in file.integrations
```

Comment:
I think we can even do
```suggestion
	some name, {"software": software} in file.integrations
```
but what have here is OK, too, of course. just code golfing.

Reply:
💥 you can't -- there are extra keys. no destructuring/pattern matching, here, unfortunately 😭

Reply:
That did not seem to work for me. Wouldn't the value need to match exactly?